src/commands/generate/serviceGenerator.ts

```diff
@@ -165,6 +165,13 @@ function generateEndpointBody(
         }
     });
 
+    const pathParamsFromPath = parsePathParamsFromPath(endpointDefinition.httpPath);
+    pathParamsFromPath.forEach(pathParam => {
```

Comment:
Since we rely on the IR generator to perform most validation we don't need this

Reply:
oh cool - good point - removed in d1c37162225c85a435e5b8c57da898f8443bdd3b